pkg/bundle/controller.go

```diff
@@ -148,18 +142,8 @@ func AddBundleController(
 		// Reconcile Bundles who reference a modified source Secret.
 		Watches(&corev1.Secret{}, b.enqueueRequestsFromBundleFunc(
 			func(obj client.Object, bundle trustapi.Bundle) bool {
-				for _, source := range bundle.Spec.Sources {
-					if source.Secret == nil {
-						continue
-					}
-
-					if source.Secret.Selector != nil {
```

Comment:
Is it ok to not check `source.Secret.Selector != nil` anymore?

Reply:
Isn't this handled within `sourceSelectsObject` function now? Or have I overlooked something?